src/FakeItEasy/ExceptionMessages.cs

```diff
@@ -28,6 +28,8 @@ internal static class ExceptionMessages
 
         public static string ArgumentConstraintCannotBeNestedInArgument => "An argument constraint, such as That, Ignored, or _, cannot be nested in an argument.";
 
+        public static string DelegateCannotCallBaseMethod => "Can not configure a delegate proxy to call base method.";

```

Comment:
I know this is the wording we've had up to now, but now that I'm paying attention to it, it hits the ear funny. How do you feel about "Can not configure a delegate proxy to call base method**s**."?

Reply:
Why plural? It's used on a specific call, so it would only call one base method (if it made sense for a delegate). Perhaps you're confusing with `FakeOptionsExtensions.CallsBaseMethods`?

Reply:
I wasn't confusing the calls. I'm going to say "grammar". I would think it would sound funny in French too. In English, I'd say either "it can't call **a** base method" or (equivalently) "it can't call base method**s**". I'd be content with either phrasing, unless you are stuck on the current one.

Reply:
> "it can't call **a** base method"

👍